src/package-install-scripts.js

```diff
@@ -279,18 +301,15 @@ export default class PackageInstallScripts {
       workQueue.add(pkg);
     }
 
+    const set = this.reporter.activitySet(
+      installablePkgs,
+      Math.min(installablePkgs, this.config.childConcurrency, workQueue.size),
```

Comment:
Also fixed a minor issue here when you only had 1 install script but you ended up seeing a bunch of `[1/-] waiting...` messages.